### PR TITLE
Firyuza Class

### PR DESCRIPTION
**Pull request by Firyuza12**
Please merge

### CODE REVIEW

File: src/develop/Firyuza.java [hunk 1]
```diff
@@ -0,0 +1,9 @@
+package develop;
+
+import org.w3c.dom.ls.LSOutput;
+
+public class Firyuza {
+    public static void main(String[] args) {
+        System.out.println("Hello World");
+    }
```
Comment (srestr53) [inline src/develop/Firyuza.java:13]: Hello Firuza , make a firuza method and don't forget to make static

**Comment by srestr53** [comment]
hello Firyuza I leave a comment on your code , please review

**Comment by srestr53** [comment]
Hello Firuza check your comments

**Review by srestr53 — CHANGES_REQUESTED** [review]
second comment

File: src/develop/Firyuza.java (hunk 1 above)
Comment (Firyuza12) [reply src/develop/Firyuza.java:13]: i edit it

**Review by srestr53 — APPROVED** [review]
Okay. to go . I'm going to merge it to develop branch

**Pull request merged**